demos/select.html

```diff
@@ -69,13 +69,13 @@
           <span class="mdc-select__selected-text">Pick a food group</span>
           <div class="mdc-simple-menu mdc-select__menu">
             <ul class="mdc-list mdc-simple-menu__items">
-              <li class="mdc-list-item" role="option"  aria-disabled="true">
+              <li class="mdc-list-item" role="option" aria-disabled="true">
                 Pick a food group
               </li>
               <li class="mdc-list-item" role="option" tabindex="0">
                 Bread, Cereal, Rice, and Pasta
               </li>
-              <li class="mdc-list-item" role="option"  tabindex="0">
+              <li class="mdc-list-item" role="option" aria-disabled="true" tabindex="-1">
```

Comment:
It just depends if we want users to be able to tab to disabled options/menu items